sqlobject/src/main/java/org/jdbi/v3/sqlobject/mixins/SqlObjectMixin.java

```diff
@@ -16,20 +16,34 @@
 import org.jdbi.v3.core.Handle;
 import org.jdbi.v3.core.HandleCallback;
 import org.jdbi.v3.core.HandleConsumer;
+import org.jdbi.v3.core.HandleSupplier;
 
 /**
- * A mixin interface to define a method for obtaining the Handle attached to a sql object
+ * SqlObject base mixin interface.  Custom mixins may be written as subinterfaces
+ * of this class with default methods.
  */
-public interface GetHandle
+public interface SqlObjectMixin<This extends SqlObjectMixin<This>>
```

Comment:
Per my comment belong on `getThis()`, I don't think this interface should have the `This` generic parameter.